workers/unity/Assets/Fps/Scripts/Config/MapBuilder.cs

```diff
@@ -442,5 +445,31 @@ public static Vector3 GetWorldDimensions(int layers)
 
             worker.LevelInstance = levelInstance;
         }
+
+        private class SceneScope : IDisposable
+        {
+            private readonly Scene activeScene;
+
+            public SceneScope(Scene newActive)
+            {
+#if UNITY_EDITOR
+                if (Application.isPlaying)
```

Comment:
why do we need that? is `Application.isPlaying` only available in the editor?

Reply:
No, but there is no reason to do this test in standalone builds

Reply:
but that code just doesn't look nice :(